juniper/src/schema/meta.rs

```diff
@@ -165,6 +165,14 @@ pub struct Field<'a, S> {
     pub deprecation_status: DeprecationStatus,
 }
 
+impl<'a, S> Field<'a, S> {
+    /// Returns true if the type is built-in to GraphQL.
+    pub fn is_builtin(&self) -> bool {
+        // "used exclusively by GraphQL’s introspection system"
```

Comment:
This seems a bit hacky, `__` is not reserved so users could add their own objects/ fields/arguments/...

We could add a `builtin: bool` flag to the relevant schema types.

Reply:
From https://github.com/graphql/graphql-spec/blob/master/spec/Section%203%20--%20Type%20System.md#schema:

"All types and directives defined within a schema must not have a name which begins with {"__"} (two underscores), as this is used exclusively by GraphQL's introspection system."